### DISCUSSION TITLE
Unable to recover github acccount no receovery codes saved

### DISCUSSION THREAD

**Discussion by Admingitjyo** [community/community#46027] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Unable to access my professiona gitgub account ....Unable to recover github account no recovery  codes saved

**Comment by radar-blip** [comment, marked as answer]
For that, you may need to contact [support](https://support.github.com/request).

**Comment by cehfisher** [comment]
Hi @Admingitjyo - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to a different category? Thanks!
